Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:16.12.0-bullseye
```

Comment:
The resulting image is 1.25GB.

Consider using slimmer base images such as the `-bullseye-slim` variant. `-alpine` is great too but that base OS differs a lot from usual distros (eg. musl libc), no need to go that far.

I also don't see much value in yarn these days; npm performs very well in 2021 and is bundled in node images.

Image sizes for comparison:  
![Screenshot_20211029_202013](https://user-images.githubusercontent.com/4985725/139483558-0be1cf15-182e-46df-b9ea-b9ecc5a111aa.png)  
`latest` is current Dockerfile; `slim` is just replacing the base image by the `-bullseye-slim` variant; `slim-npm` also uses npm instead of yarn ([Dockerfile](https://github.com/ppy/scthumber/files/7444161/Dockerfile.txt), don't forget to generate package-lock.json beforehand). 
Good practice is to also set `NODE_ENV` to production.